### PR TITLE
Release rancher-turtles 0.2.0+up0.11.0

### PR DESCRIPTION
**Pull request by hardys**
This rebases to the latest 0.11.0 turtles release, and also rebases the versions of Core CAPI, CAPM3 and the RKE2 provider.

Note that to enable this to install reliably via EIB we currently need a workaround for https://github.com/rancher/turtles/issues/627 until a version with https://github.com/rancher/turtles/pull/705 is released, passing a values file like this should work:

```
rancherTurtles:
  features:
    embedded-capi:
      disabled: false
```

It's then also necessary to pass a manifest as follows since the chart pre-install hook is no longer taking care of this.

```
apiVersion: management.cattle.io/v3
kind: Feature
metadata:
  name: embedded-cluster-api
spec:
  value: false
```

### CODE REVIEW

**Review by alknopfler — APPROVED** [review]
LGTM

**Comment by hardys** [comment]
Setting this back to draft as @Danil-Grigorev pointed out that the repository override we used via clusterctl isn't working via the CAPI operator: 

```
> kubectl get deployment -n capi-system -o yaml | grep image
          image: registry.k8s.io/cluster-api/cluster-api-controller:v1.7.5
          imagePullPolicy: IfNotPresent
> kubectl get deployment -n rke2-bootstrap-system -o yaml | grep image
          image: ghcr.io/rancher/cluster-api-provider-rke2-bootstrap:v0.6.1
          imagePullPolicy: IfNotPresent
```

I will try to find a different approach via the providers section

**Comment by Danil-Grigorev** [comment]
The scenario with repository `images` overrides via `clusterctl.yaml` will be working once https://github.com/kubernetes-sigs/cluster-api-operator/pull/598 is merged and released.

**Comment by hardys** [comment]
Ok this is now working with all images overridden to the OBS ones:

```
> for ns in capi-system capm3-system rke2-bootstrap-system rke2-control-plane-system; do kubectl get deployment -n $ns -o yaml | grep "image:"; done
          image: registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-controller:1.7.5
          image: registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-metal3:1.7.1
          image: registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/ip-address-manager:1.7.1
          image: registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-rke2-bootstrap:0.6.1
          image: registry.opensuse.org/isv/suse/edge/clusterapi/containers/images/cluster-api-provider-rke2-controlplane:0.6.1

```

@alknopfler and @Danil-Grigorev please could you take another look and check if you're happy with this revision?  Thanks!

**Review by Danil-Grigorev — APPROVED** [review]
This looks good to me, just a comment - the original issue should be fixed and we are planning to release it soon.

File: charts/rancher-turtles/0.2.0+up0.11.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  catalog.cattle.io/release-name: rancher-turtles
+  catalog.cattle.io/scope: management
+  catalog.cattle.io/type: cluster-tool
+apiVersion: v2
+appVersion: 0.11.0
+dependencies:
+- condition: cluster-api-operator.enabled
+  name: cluster-api-operator
+  repository: file://./charts/cluster-api-operator
+  version: 0.12.0
```
[the remark this answers is not in the document]
Comment (hardys) [reply charts/rancher-turtles/0.2.0+up0.11.0/Chart.yaml:18]: Thanks a lot for working on that - we can pick up that in our next rebase which should simplify the configuration 👍

**Pull request merged**